lua/lazyvim/plugins/extras/ai/codeium.lua

```diff
@@ -30,4 +44,18 @@ return {
       table.insert(opts.sections.lualine_x, 2, LazyVim.lualine.cmp_source("codeium"))
     end,
   },
+
+  {
+    "saghen/blink.cmp",
+    optional = true,
+    opts = {
+      sources = {
+        compat = vim.g.ai_cmp and { "codeium" } or nil,
+      },
+    },
+    dependencies = {
+      "codeium.nvim",
+      vim.g.ai_cmp and { "saghen/blink.compat" } or {},
```

Comment:
@folke Please take a look at [this comment](https://github.com/Saghen/blink.compat/pull/6#discussion_r1828128222). The maintainer of `blink-compat` says that it should not be in `blink.cmp`'s dependencies. And when I tested it, I also put it as a plugin spec on its own, not a dependency. Not sure if it actually affects anything, just thought I should let you know about this.

Reply:
It doesn't really matter since it would be loaded directly anyway.
`blink-compat` is a library, so adding it there doesn't really load anything.

Reply:
Whatever you think is best. Just wanted to make you aware of this and let you be the judge, since you know way better compared to my novice knowledge.